syntax = "proto3";

package facerecog;

message IdentifyRequest {
    string image_path = 1;
}

message IdentifyResponse {
    int32 person_id = 1;
}

service Identify {
    rpc Identify(IdentifyRequest) returns (IdentifyResponse) {}
}
